lib/robotis.ex

```diff
@@ -63,9 +85,10 @@ defmodule Robotis do
   def init(opts) do
     port = Keyword.fetch!(opts, :uart_port)
     baud = Keyword.get(opts, :baud, 57_600)
+    control_table = Keyword.get(opts, :control_table, :xl330_m288)
```

Comment:
I think we don't need to default to anything, and instead we should make the control_table a required option.  What do you think?